sdk/python/core/samples/path_bgp.py

```diff
@@ -34,7 +35,7 @@ def execute_path(provider, codec):
 
 
 if __name__ == "__main__":
-    repo = Repository("/var/folders/jt/kfhqscp54dq3gfwdpkbj48zm0000gn/T//127.0.0.1:12022")
-    provider = RestconfServiceProvider(repo, '127.0.0.1', 'admin', 'admin', 8008)
-    codec = CodecService()
+    repo = Repository("/Users/abhirame/.ydk/127.0.0.1:12022")
```

Comment:
Possibly. Maybe I should remove the `repo` and have the sample download the models